common/components/chrome/SubHeader.jsx

```diff
@@ -49,9 +50,7 @@ class SubHeader extends React.Component<{||}, State> {
       <div className={this._cx.get('root')}>
         {this._renderSectionLinks()}
         <span className={this._cx.get('rightContent')}>
-          <span className={this._cx.get('createProject')} onClick={this.navigateToCreateProject.bind(this)}>
-            Create A Project
-          </span>
+          {this._renderCreateProjectButton()}
```

Comment:
I'd hoist the `shouldRender` logic up into this function so that it's more obvious from reading the component's render method that this button may not always be visible in the UI. For example, this line would become

```
{
  CurrentUser.isLoggedIn() ? this._renderCreateProjectButton() : null
}
```

And then below we'd just have

```
_renderCreateProjectButton(): React$Node {
  return <span>...</span>
}
```

Reply:
Done